packages/wallet/.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+const rules = {
+  "arrow-body-style": "error",
+  "arrow-parens": ["off", "as-needed"],
+  camelcase: "error",
```

Comment:
Can we be consistent with using quoted keys versus non-quoted keys?

Reply:
Yes thanks